tests/Functional/ContentNewsRenderTest.php

```diff
@@ -140,6 +140,15 @@ public function testNewsRendering(): void {
     $this->assertContentHeader($contacts_content, 'Contacts');
     $this->assertContactDefaultRender($contacts_content, 'news_contact');
 
+    // Add a different and unpublished media and assert it is not rendered
+    // in the contact.
+    $contact_media = $this->createMediaImage('news_contact_media');
```

Comment:
Well, this would not assert the caching. You should retrieve the media that was created for that default contact, unpublish it, and then assert it is no longer rendered.